package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "verus-typescript-primitives",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Don't worry about changing the version in the PR, I'll rebase the PR into a new branch and change the version when it goes to master